src/selectors.js

```diff
@@ -18,7 +18,7 @@ export type Selection<T> = {
   data?: T,
   isLoading: boolean,
   needsFetch: boolean,
-  error?: Error | { message: string },
+  error?: Error,
```

Comment:
This is the biggest change. I wonder if there should be even more severe checks to enforce this.